tests/brun/concat-4.txt

```diff
@@ -1,2 +1,2 @@
 brun -c '(concat (q foo) (q (bar)))'
-FAIL: concat on list ("bar")
+FAIL: concat on list ("foo" ("bar"))
```

Comment:
These lists aren't being deserialized correctly.

Reply:
hm. both rust and python prints them like this, as if it's deliberate. @richardkiss ?

Reply:
I don't understand the objection. The arguments to `concat` are exactly `("foo" ("bar"))` in this example.